### ISSUE TITLE
Host meeting media

### ISSUE THREAD

**Issue by csarven** [solid/solidproject.org#791]
In order to assert essential authority over Solid community's media assets, in particular video and audio from meetings, the assets should be published under its own URI spaces. Likely under solidproject.org or solidcommunity.net, e.g.:

`https://solidproject.org/media/{yyyy}/{mm}/{dd}/{file}'

The URI space should a persistence policy. Either extend https://solidproject.org/terms#uri-persistence-policy or have its own policy (wherever it ends up).

It should be archivable by the Internet Archive / Wayback machine.

Media files can be pushed through GitHub (if no filesize limitations?) or Solid server ( https://github.com/solid/solidproject.org/issues/248 ).

---

For example, here is a very rough estimate from a Solid CG meeting recording in Jitsi
<details>
<summary>More details of ~150MB webm file, duration 20 minutes</summary>
<dl>
<dt>Input #0, matroska,webm, from 'solid-cg_2023-06-21T14_38_36.268Z.webm':</dt>
<dt>Metadata:</dt>
<dd>encoder         : Chrome</dd>
<dt>Duration:</dt> 00:19:01.58, start: 0.000000, bitrate: 1043 kb/s</dt>
<dt>Stream #0:0(eng):</dt> Audio: opus, 48000 Hz, stereo, fltp (default)</dd>
<dt>Stream #0:1(eng):</dt><dd>Video: h264 (Constrained Baseline), yuv420p(tv, bt709, progressive), 1670x1078, SAR 1:1 DAR 835:539, 60 tbr, 1k tbn (default)</dd>
</dl>
</details>

 Putting any optimisations aside, we could expect 500MB for a 60 minute meeting. Say 26GB of diskspace for 52 meetings. I'm not sure what the bandwidth requirements would be at this time but we don't need anything too fancy, so just hosting raw files is fine.

What may be other technical requirements? What may be the costs?

**Comment by elf-pavlik** [comment]
Whenever it gets hosted in solid storage, I would like to create 360p, 480p, and 720p variants of the video, this way it could help us to resolve issues like:

* https://github.com/solid/data-interoperability-panel/issues/33

TL;DR Someone linking to the video shouldn't even think about choosing some specific resolution, and anyone following the link to the video should be able to discover all the available variants.

Of course, I would take care of transcoding all the variants.
